packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -132,13 +132,58 @@ export const css = curry(async function css(
     true
   )
 
+  function getModuleStyleLoader(
+    cssOptions: webpack.ParserOptions,
+    preProcessor: string
+  ) {
+    const loaders: webpack.RuleSetUseItem[] = [
+      // Resolve CSS `@import`s and `url()`s
+      {
+        loader: require.resolve('css-loader'),
+        options: cssOptions,
+      },
+
+      // Compile CSS
+      {
+        loader: require.resolve('postcss-loader'),
+        options: {
+          ident: '__nextjs_postcss',
+          plugins: postCssPlugins,
+          sourceMap: true,
+        },
+      },
+    ].filter(Boolean)
+
+    // Add appropriate development mode or production mode style loader first
+    if (ctx.isClient) {
+      loaders.unshift(
```

Comment:
Unshift is weird here, I'd rather see it come first (in the correct order).

Reply:
The original loader implementation from built-in CSS used a wrapped expression to evaluate whether to include `getClientStyleLoader()`:

```
ctx.isClient &&
  getClientStyleLoader({
    isDevelopment: ctx.isDevelopment,
    assetPrefix: ctx.assetPrefix,
  }),
```

This had to be filtered and us TS `as` in order to compile.

At: https://github.com/zeit/next.js/blob/canary/packages/next/build/webpack/config/blocks/css/index.ts#L215

The `unshift()` allows you to avoid both. I can mimic the original implementation if desired.

Reply:
No need to mimic original, let's make loaders start as an empty array and push within an `if (...) { ... }` block before the others are pushed.